src/node/node_class.hpp

```diff
@@ -49,6 +49,8 @@ class ObjectWrap : public Nan::ObjectWrap {
         return Nan::New(js_template);
     }
 
+    static void construct(const v8::FunctionCallbackInfo<v8::Value>&);
```

Comment:
Instead of hardcoding this to v8, we should instead get these types from the `Types` metatype. This way we can reuse all of this code for v8 and nan by just changing the `Types` type.